src/test/java/gov/healthit/chpl/aqa/asserts/ChplSearchPageAsserts.java

```diff
@@ -194,4 +194,16 @@ public void verifyCertStatusOptionUnchecked(final String certStatusFilterOption)
         ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
         assertTrue(!(SearchPage.filterOption(getDriver(), certStatusFilterOption).isSelected()));
     }
+
+    /**
+     * Assert that complaints reporting menu navigation text is correct in Surveillance drop-down.
+     * @param expectedMenuNavigationText - Complaints Reporting
+     */
+    @Then("^I see \"([^\"]*)\" as the menu navigation text for Complaints Reporting$")
+    public void iSeeComplaintsReportingMenuNavigationText(final String expectedMenuNavigationText) {
+        WebElement link = SearchPage.complaintsReporting(getDriver());
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].scrollIntoView();", link);
```

Comment:
Is this actually necessary? I don't know that the elements need to be in view to get their text, and I'm not even positive that they actually scroll into view, as I think they just scroll into the viewport, and when they do that they're often hidden behind the sticky header or footer

Reply:
Nope, it's not necessary in this case!

Reply:
Did you check the other places where this "scrollintoview" thing happens?

Reply:
Yes, I did and it works for the other scenarios! For example:- I have used it while adding an Announcement

Reply:
Sorry, I see what you mean and yes I removed it from the other step as well